snippets/oidc-cloudfront.adoc

```diff
@@ -0,0 +1,12 @@
+
+//This snippet appears in the following assemblies:
+//
+// * ../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc
+
+:_content-type: SNIPPET
+[NOTE]
+====
+As of ROSA CLI 1.2.8, all new clusters have a new format for the OIDC provider endpoint URL. {product-title} cluster OIDC provider URLs are no longer regional. The AWS CloudFront implementation provides improved access speed and resiliency as well as reduces latency.
```

Comment:
If I'm not wrong we have the new format for OIDC provider endpoint URL from **v1.2.7**.
`oadler@dhcp-4-86:rosa ((7fc46054...))$ ./rosa version
1.2.7
oadler@dhcp-4-86:rosa ((7fc46054...))$ ./rosa create cluster ...
...
OIDC Endpoint URL:          https://dvbwgdztaeq9o.cloudfront.net/20f03tas19t16ktu1us0re9pcoehr11c`